pkgs/development/python-modules/ifcfg/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, fetchPypi
+, buildPythonApplication
+, iproute2
+}:
+
+buildPythonApplication rec{
+  pname = "ifcfg";
+  version = "0.22";
```

Comment:
0.23 released

Reply:
Updated.

Reply:
Please update version in PR and commit title :)